src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -14,6 +14,8 @@ namespace System.Net.Http
     internal class WinInetProxyHelper
     {
         private const int RecentAutoDetectionInterval = 120_000; // 2 minutes in milliseconds.
+        private readonly string _autoConfigUrl, _proxy, _proxyBypass;
```

Comment:
What's the motivation for removing auto-properties here?

Reply:
Testing. I need to set these via reflection to get a proper functional test.

Read-only props can't be set via reflection, and there is no good way to find the backing field for an auto-prop.